templates/taxi/car_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  {% if object %}
+    <h1>Update car "{{ car.model }}"</h1>
+  {% else %}
+    <h1>Add new car to service</h1>
+  {% endif %}
+
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input type="submit" value="{{ object|yesno:"Update, Add" }}">

```

Comment:
By the way, this button (and all buttons like this in your project) is not styled the same as other buttons on your website. Why don't you use bootstrap styling for buttons? You **can** do it for `<input>` elements.